docs/users/installation.md

```diff
@@ -181,34 +183,17 @@ configurations. To enable Scalafix for other configuration like
 
 ### Multi-module builds
 
-The `scalafix` task aggregates like the `compile` and `test` tasks. To run
-Scalafix on all projects for both main and test sources you can execute
-`all scalafix test:scalafix`.
-
-Optionally, add a command alias to your build to run Scalafix on your entire
-project with the shorthand `fix`
-
-```scala
-// top of build.sbt
-addCommandAlias("fix", "all compile:scalafix test:scalafix")
-```
+Both the `scalafix` & the `scalafixAll` task aggregate like the `compile`
+and `test` tasks. To run Scalafix on all projects for both main and test
+sources you can execute `all scalafix test:scalafix` or simply `scalafixAll`.
```

Comment:
This seems like the easiest to explain option and it handles custom integrations like `it:scalafix`
```suggestion
sources you can execute `scalafixAll`.
```

Is there a difference between `all scalafixAll` and only `scalafixAll`?

Reply:
indeed :+1: 

> Is there a difference between all scalafixAll and only scalafixAll?

None as in both cases, configurations invocations will run independently/concurrently. I guess the latter allows a single `last scalafixAll` to see an overview of the logs easily. And the former is a command while the latter is a task obviously.  Why do you ask?